JS/Lesson26/onur/createNewPost/styles.css

```diff
@@ -0,0 +1,70 @@
+* {
```

Comment:
A suggestion for your future projects: for generic CSS that are added for each page of your site/application, create a main.CSS file where you keep these styles. Then, just add that file to each HTML page. Only add a new CSS file that contain specific styles that are only applicable to this specific page.